AdminWebsite/AdminWebsite/Models/EditHearingRequest.cs

```diff
@@ -44,6 +44,8 @@ public EditHearingRequest()
         /// </summary>
         public List<EditParticipantRequest> Participants { get; set; }
 
+        public List<EditTelephoneParticipantRequest> TelephoneParticipants { get; set; }
+
```

Comment:
Do we need this new EditTelephoneParticipantRequest object altogether to append existing EditParticipantRequest object with a new ParticipantType ( to indicate it is a Video/Telephone participant) ?

Reply:
yes, Telephone participants are holding in a different model because it has fewer properties. When I was using the participant some of the fields that are not there in telephone participant are mandatory in the participant request.